database/src/main/java/database/exception/IdMismatchException.java

```diff
@@ -0,0 +1,8 @@
+package database.exception;
+
+public class IdMismatchException extends BadRequestException {
```

Comment:
Not used, please remove.

Reply:
Now it used in handleUpdateRecord method.